src/main/java/mortalreminder/io/MortalReminderWindow.java

```diff
@@ -0,0 +1,69 @@
+package mortalreminder.io;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import mortalreminder.MortalReminder;
+import mortalreminder.commands.Command;
+import mortalreminder.commands.CommandTypes;
```

Comment:
Import statements are ordered consistently, a nit would be to separate the mortal reminder and javafx imports.